src/components/credit-card.jsx

```diff
@@ -0,0 +1,28 @@
+function CreditCard(card) {
+    const divStyle = { backgroundColor: card.bgColor, color: card.color, width: "315px", height: "165px" };
+    const imgStyle = {};
+    
+    let logo;
+
+    card.type === "Visa" ? logo = "https://logodownload.org/wp-content/uploads/2016/10/visa-logo.png" : logo = "https://logodownload.org/wp-content/uploads/2014/07/mastercard-logo-3.png";
+    card.type === "Visa" ? imgStyle.width = "45px" : imgStyle.width = "35px";
+    card.type === "Visa" ? imgStyle.height = "20px" : imgStyle.height = "20px";
```

Comment:
```suggestion
   imgStyle.height = "20px" ;
```
Salvo que me digas lo contrario en la ternaria estas poniendo lo mismo jejejejeje